its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/Tests.java

```diff
@@ -73,4 +74,56 @@ private static Orchestrator createOrchestrator() {
     }
     return orchestrator.build();
   }
+
+  private static String getPluginVersion(String runtimeVersion, String pluginKey) {
+    return System.getProperty(pluginKey, getDefaultPluginVersion(runtimeVersion, pluginKey));
+  }
+
+  private static String getDefaultPluginVersion(String runtimeVersion, String pluginKey) {
```

Comment:
This is not the correct way, it should be done in the YML like this:
https://github.com/SonarSource/sonar-scanner-msbuild/blob/e8e1379419222d45255b6aab50ea1c0abce4be48/azure-pipelines.yml#L355-L357

I've recently cleaned up a mess like this for the XML analyzer, as it was causing quite some troubles.